src/tools/runk/libcontainer/src/container.rs

```diff
@@ -358,6 +364,33 @@ pub fn create_linux_container(
     Ok(container)
 }
 
+// Load rustjail's Linux container.
+// "uid_map_path" and "gid_map_path" are always empty, so they are not set.
+pub fn load_linux_container(
+    status: &Status,
+    console_socket: Option<PathBuf>,
+    logger: &Logger,
+) -> Result<LinuxContainer> {
+    let mut container = LinuxContainer::new(
+        &status.id,
+        &status
+            .root
+            .to_str()
+            .map(|s| s.to_string())
+            .ok_or_else(|| anyhow!("failed to convert a root path"))?,
+        status.config.clone(),
+        logger,
+    )?;
+    if let Some(socket_path) = console_socket.as_ref() {
+        container.set_console_socket(socket_path)?;
+    }
+
+    container.init_process_pid = status.pid;
+    container.init_process_start_time = status.process_start_time;
+    container.created = status.created.into();
```

Comment:
```diff
- container.cgroup_manager = Some(status.cgroup_manager.clone());
```

I removed the copy of the `cgroup manager`.
The cgroup manager retrieved from the status file does not contain the `cgroup` field due to `#[serde(skip)]` and a new exec process cannot launch properly inside the exisiting namespace.
We can remove the copy safely because the `LinuxContainer::new()` can create a proper `cgroup_manager`.

Reply:
@ManaSugi  I got it. Thank you!